### ISSUE TITLE
Index in root

### ISSUE THREAD

**Issue by reagle** [ennioitaliano/obsidian-auto-glossary#19]
Hi! Thank you for your plugin. I'm looking for a way to create a single index page for a vault, in the root folder, show it and all its subfolder pages.  Something like what Zim creates. Can this do that? I haven't been able to figure it out if so.

**Comment by ennioitaliano** [comment]
No, actually there is no way to do that with my plugin, and even if you could pick the root folder your subfolders wouldn't be handled well. I will try to implement this feature and to handle subfolders better in the next updates.

**Comment by ennioitaliano** [comment]
Vault-wide index generation in the root folder is fully supported in v1.0.0 (by right-clicking the vault root folder or using the Command Palette command `Auto Glossary: Create index or glossary`). For hierarchical tree rendering by subfolders, please track #11.
